Patches/0001-ShellPkg-Add-support-for-input-with-separately-repor.patch

```diff
@@ -6,12 +6,12 @@ Subject: [PATCH 1/1] ShellPkg: Add support for input with separately reported
 
 REF: https://bugzilla.tianocore.org/show_bug.cgi?id=2510
 
-Some firmwares:
+Some types of firmware:
 - Report Shift modifier even when they report upper-case unicode letter.
```

Comment:
You cannot edit patch files as they affect the files we do not own.

Reply:
Yes, Should not ... at least not to items not specifically being added at most.